lib/runtime-c-api/build.rs

```diff
@@ -14,18 +14,21 @@ fn main() {
 
     let mut pre_header = r#"
 #if !defined(WASMER_H_MACROS)
+
 #define WASMER_H_MACROS
 
-#if defined(MSVC)
-#if defined(_M_AMD64)
-#define ARCH_X86_64
-#endif
+// Define the `ARCH_X86_X64` constant.
+#if defined(MSVC) && defined(_M_AMD64)
+#  define ARCH_X86_64
```

Comment:
I thought the `#` had to be touching the `define` 🤔

Reply:
To clarify, the `#` before and after the `#` are Rust syntax for raw strings (we can use any number of `#` if the string `"#` appears inside the string).  The rest of the `#` are part of the C header file

Reply:
It's fine from a rust perspective, I just thought it was strange C -- thanks for the clarification!

Reply:
Putting spaces before `define` helps to provide some visual structure to the block.

Marking the conversation as resolved :-)!